src/test/java/tr/com/srdc/cda2fhir/IntegrationTest.java

```diff
@@ -81,8 +84,21 @@ public void rakiaIntegration() throws Exception {
 		Assert.assertEquals(1, patientResults.getTotal());
 		Assert.assertEquals(18, practitionerResults.getTotal());
 		Assert.assertEquals(14, medicationResults.getTotal());
+	}
 
+	@Test
+	public void provenanceIntegration() throws Exception {
+		String sourceName = "Cerner/Person-RAKIA_TEST_DOC00001 (1).XML";
+		String encodedBody = Base64.getEncoder().encodeToString("<ClinicalDoc>\n</ClinicalDoc>".getBytes());
+		Identifier device = new Identifier();
+		device.setValue("Higgs");
+		device.setSystem("http://www.amida.com");
 
+		Bundle transactionBundle = ccdTransformer.transformDocument("src/test/resources/" + sourceName,
+				BundleType.TRANSACTION, null, encodedBody, device);
+
+		// print pre-post bundle
+		FHIRUtil.printJSON(transactionBundle, "src/test/resources/output/provenance_bundle.json");
```

Comment:
Is there any assertions at all in this test? As it is it looks like it will only test crashes. I would put asserts on  device and encodedBody you are sending.

Reply:
Done.